app/templates/welcome.js

```diff
@@ -23,11 +23,11 @@ module.exports = function(state, emit) {
         'uploadPageSizeMessage'
       )}</em></span>
       <form method="post" action="upload" enctype="multipart/form-data">
+        <input id="file-upload" type="file" name="fileUploaded" onchange=${upload} onfocus=${onfocus} onblur=${onblur} />
```

Comment:
The html attributes onfocus and onblur have the same issues as the css in Firefox, but it seems I can use them in the template as they get rewritten into javascript event listeners? (which do work in Firefox). I clearly need to read some documentation to understand this better, will it _always_ remove & replace the html onfocus attribute? (becuase if it doesn't, this will brake).

Reply:
It seems to be working for me now. 👍 
Can you re-base so this can merge smoothly?